src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -780,6 +780,11 @@ public void UpdateConfigurations(DatabaseRecord databaseRecord)
             var previousConfiguration = existingBackupState.Configuration;
             existingBackupState.Configuration = newConfiguration;
 
+            if (previousConfiguration.BackupType != newConfiguration.BackupType)
+            {
```

Comment:
Please add a comment that we clear the in-memory state here

Reply:
done